lib/policy_machine_storage_adapters/neography.rb

```diff
@@ -163,7 +163,7 @@ def element_in_machine?(pe)
     ##
     # Add the given association to the policy map.  If an association between user_attribute
     # and object_attribute already exists, then replace it with that given in the arguments.
-    def add_association(user_attribute, set_of_operation_objects, object_attribute, policy_machine_uuid)
+    def add_association(user_attribute, object_attribute, policy_machine_uuid = nil)
```

Comment:
We don't use pm uuid in associations. I got rid of it but decided that change was too big for this PR, so I reverted. Missed a reversion.